src/xcode/ENA/ENA/Source/Services/HealthCertificate/HealthCertificateService.swift

```diff
@@ -302,6 +274,28 @@ class HealthCertificateService {
 		recycleBin.moveToBin(.certificate(healthCertificate))
 	}
 
+	func checkForCCLConfigurationAndRulesUpdates(completion: (() -> Void)? = nil) {
+		cclService.updateConfiguration { [weak self] didChange in
+			guard let self = self else { return }
```

Comment:
```suggestion
			guard let self = self else { 
			completion?()
			return 
			}
```

Just to be on the safe side.